src/main/java/com/socialbox/controllers/GroupController.java

```diff
@@ -48,7 +48,13 @@ public Group saveGroup(@RequestBody Group group) {
   }
 
   @GetMapping("/invite")
-  public InviteDTO sendInvite(@RequestParam String groupId,@RequestParam String userId){
-    return this.groupService.sendInvite(groupId, userId);
+  public ResponseEntity<InviteDTO> sendInvite(@RequestParam String groupId, @RequestParam String userId) {
+    InviteDTO inviteDTO = this.groupService.sendInvite(groupId, userId);
+
+    if (inviteDTO == null) {
+      return ResponseEntity.badRequest().build();
```

Comment:
Need to have clarity whether the response is a success or not, not wrapping our DTOs with ResponseEntity will give a single Status to all the cases.